components/Challenge.yml

```
@@ -0,0 +1,37 @@
+openapi: 3.0.0
+info:
+  version: "0.0.3-unstable"
+  title: VC API
+  description: This is an Experimental Open API Specification for the [VC Data Model](https://www.w3.org/TR/vc-data-model/).
+  license:
+    name: W3C Software and Document License
+    url: http://www.w3.org/Consortium/Legal/copyright-software.
+  contact:
+    name: GitHub Source Code
+    url: https://github.com/w3c-ccg/vc-api
+paths:
+components:
+  schemas:
+    CreateChallengeResult:
+      type: object
+      additionalProperties: false
+      description: Object containg a challenge
+      properties:
+        challenge:
+          type: string
+          description: The challenge value
+    ChallengeVerificationMetadata:
+      type: object
+      additionalProperties: false
+      description: Metadata about the verification of options.challenge.
+      properties:
+        verified:
+          type: string
+          description: Whether verification of the challenge was successful
+        firstVerified:
```

Do we want this property name to better communicate that its value is a date-time vs. something else?

```suggestion
        firstVerifiedAt:
```